arches_component_lab/src/arches_component_lab/types.ts

```diff
@@ -1,13 +1,23 @@
+export interface AliasedTileNodeValue {
+    display_value: string;
+    node_value: unknown;
+    details: unknown[];
+}
+
 export interface CardXNodeXWidget {
     card: {
         name: string;
     };
-    config: unknown;
+    config: {
+        defaultValue: unknown | null;
+        placeholder: unknown | null;
```

Comment:
not every widget defines a placeholder (eg: radio-boolean, switch, geom, etc..).  I don't think we should hardcode this in here.